nautobot/core/templatetags/helpers.py

```diff
@@ -43,7 +43,7 @@
 
 @library.filter()
 @register.filter()
-def hyperlinked_object(value, field="display"):
+def hyperlinked_object(value, field="display", **kwargs):
```

Comment:
I think it would be preferable to specifically enumerate `target` and `rel` as kwargs rather than having a catchall kwargs that could be used to provide any arbitrary HTML attribute, including unsafe ones.

Reply:
Agreed!  I was trying to accommodate future additions, but that is the better option.